test/libsolidity/smtCheckerTests/types/struct/array_struct_array_struct_memory_safe.sol

```diff
@@ -32,5 +32,6 @@ contract C {
 }
 // ====
 // SMTEngine: all
+// SMTTargets: assert
```

Comment:
Similar to above, the goal here is to test the assertions, not the array accesses.